docs/sources/setup/kubernetes.md

```diff
@@ -139,6 +139,6 @@ spec:
           env:
             - name: EXECUTABLE_NAME # Select the executable by its name instead of OPEN_PORT
               value: "goblog"
-            - name: OTEL_EXPORTER_OTLP_ENDPOINT
+            - name: BEYLA_OTEL_EXPORTER_OTLP_ENDPOINT
```

Comment:
I think this should stay `OTEL_EXPORTER_OTLP_ENDPOINT`, correct?

Reply:
Ah yeah! Thanks for realizing